### PR TITLE
Adjust mention text attachment token appearance

### PR DESCRIPTION
**Pull request by daehn**
## What's new in this PR?

* Adjust mention token rendering.
* Add `mentions` property to `MarkdownTextView`.
* User new data-model API to append mentions together with text messages.

### CODE REVIEW

File: Wire-iOS/Sources/UserInterface/Conversation/InputBar/ConversationInputBarViewController+UITextViewDelegate.swift [hunk 1]
```diff
@@ -49,6 +49,16 @@ extension ConversationInputBarViewController: UITextViewDelegate {
...
+        
+        // Start mentioning
+        if text == "@" {
+            let attachment = MentionTextAttachment(configuration: .init(user: .selfUser()))
+            let attributedString = NSAttributedString(attachment: attachment)
+            textView.attributedText = textView.attributedText + attributedString
+            return false
+        }
+
+//        mentioningInTextView(textView, range: range, replacementText: text)
```
Comment (mikeger) [inline Wire-iOS/Sources/UserInterface/Conversation/InputBar/ConversationInputBarViewController+UITextViewDelegate.swift]: Commented text, by design?

**Comment by mikeger** [comment]
@daehn I am starting on highlighting the mentions in the text, I guess I can use the `MentionsTextAttachment` for it as well? Not sure if we want to be able to break mentions in multiple lines in the posted message. If yes, then I can just style the text.

**Comment by daehn** [comment]
@mikeger I am not sure if it make sense to use it there as well as it prevents line breaks, maybe it's easier to just add a link attribute with special attribute etc. when rendering the message?

**Comment by daehn** [comment]
@mikeger Then again it might make things easier if we can reuse the attachment, but I am not sure how we can handle taps on a text attachment yet.

File: Wire-iOS/Sources/UserInterface/Conversation/InputBar/ConversationInputBarViewController+UITextViewDelegate.swift (hunk 1 above)
Comment (daehn) [reply Wire-iOS/Sources/UserInterface/Conversation/InputBar/ConversationInputBarViewController+UITextViewDelegate.swift]: @mikeger updated!

**Comment by mikeger** [comment]
I'll try just styling the text for the moment.

**Comment by mikeger** [comment]
Hey @cardassian-tailor, we are tackling the mentions now, and yes, it is related to the feature request.

**Comment by vytis** [comment]
So maybe let's merge this and continue work in other PRs?

**Pull request merged**